src/Managers/ClilocManager.cpp

```diff
@@ -277,3 +276,59 @@ wstring CClilocManager::ParseArgumentsToClilocString(int cliloc, bool toCamelCas
 
     return message;
 }
+
+wstring CClilocManager::ParseXmfHtmlArgumentsToCliloc(int cliloc, bool toCamelCase, wstring args)
```

Comment:
This is basically a copy of `ParseArgumentsToClilocString` and only the start of the function changes (the while erase part). Then to make it a bit better, do a 3rd private function with only the code after the first while, and do these two functions call the new one passing the new cleared args, unless there is any other differences between they that I'm not seeing right now.